db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] Kindly create 5 unique which will be used to test your code. I did add i in the image below, and it was working perfectly. Good job 👏 

- Kindly note my code is just a suggestion. Kindly be creative

### View:
![image](https://github.com/JuanDBta/hello-rails-react/assets/50990925/fc93b87b-39d6-48ae-8e0a-a1173fa8cb14)

### Requirement:
![image](https://github.com/JuanDBta/hello-rails-react/assets/50990925/3b9e9a47-6f8f-4989-bac0-6818cf4f7cc9)